lib/cea/mp4_cea_parser.js

```diff
@@ -129,6 +142,21 @@ shaka.cea.Mp4CeaParser = class {
           shaka.util.Error.Code.INVALID_MP4_CEA);
     }
 
+    // If the content is encrypted we have no way of knowing the
+    // bitstreamformat, so we try to get it from the mimetype.
+    if (encrypted && this.bitstreamFormat_ == BitstreamFormat.UNKNOWN) {
```

Comment:
I don't see why this would be true.  In the samples I looked at, the encv box was followed by a box like avc1.

It also appears that the encv contains other boxes which would indicate the codec (avc1).  The encv box contains a sample description, followed by a sinf box, which contains a frma box.  frma only has a 4-byte field with the name of the codec.

If we can't make that work, we can do it this way for now.  But I feel like parsing encv should be preferable.